setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='ionhash',
-    version='1.3.0-SNAPSHOT',
+    version='1.3.dev',
```

Comment:
This is the standard for a "pre-release" python package version per https://peps.python.org/pep-0440/